bot/commands/command_manager.go

```diff
@@ -153,6 +154,18 @@ func (cm *CommandManager) OnMessage(message discordgo.Message) (err error) {
 
 	commandObject, exists := cm.Commands[commandTrigger]
 	if !exists {
+		newdist := 100
+		result := "None"
```

Comment:
It would make more sense to initialise this empty `""` then after the loop, only send the suggestion if it isn't still empty.